<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainServerWindow</class>
 <widget class="QMainWindow" name="MainServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serwer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Zarządzaj serwerem</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtPort">
             <property name="toolTip">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Liczba z zakresu &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;lt;1024,&lt;/span&gt; &lt;span style=&quot; font-weight:600;&quot;&gt;65535&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="inputMask">
              <string extracomment="Liczba z zakresu: 1024-65535">99999; </string>
             </property>
             <property name="text">
              <string>44444</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Działania:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSrvReset">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Resetuj</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSrvStop">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnSrvStart">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Zarządzaj bazą danych</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Dodaj cytat:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="quoteArea">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBDAdd">
               <property name="text">
                <string>Dodaj</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableView" name="bdTable">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>100</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>50</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>388</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnDel">
               <property name="text">
                <string>Usuń zaznaczone cytat</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnBDSubmit">
               <property name="text">
                <string>Zapisz</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Serwer">
    <property name="title">
     <string>&amp;Serwer</string>
    </property>
    <addaction name="actionResetuj"/>
    <addaction name="actionStop"/>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionOtw_rz_plik_log_w_serwera"/>
    <addaction name="separator"/>
    <addaction name="action_Zako_cz"/>
   </widget>
   <widget class="QMenu" name="menu_Pomoc">
    <property name="title">
     <string>&amp;Pomoc</string>
    </property>
    <addaction name="actionO_programie"/>
    <addaction name="separator"/>
    <addaction name="actionResetuj_baz_danych"/>
   </widget>
   <addaction name="menu_Serwer"/>
   <addaction name="menu_Pomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionResetuj">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Resetuj</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Zako_cz">
   <property name="text">
    <string>&amp;Zakończ</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionResetuj_baz_danych">
   <property name="text">
    <string>Resetuj bazę danych</string>
   </property>
  </action>
  <action name="actionOtw_rz_plik_log_w_serwera">
   <property name="text">
    <string>Otwórz plik logów serwera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnSrvStart</tabstop>
  <tabstop>btnSrvStop</tabstop>
  <tabstop>edtPort</tabstop>
  <tabstop>btnSrvReset</tabstop>
  <tabstop>quoteArea</tabstop>
  <tabstop>btnBDAdd</tabstop>
  <tabstop>bdTable</tabstop>
  <tabstop>btnBDSubmit</tabstop>
  <tabstop>btnDel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>action_Zako_cz</sender>
   <signal>activated()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSrvReset</sender>
   <signal>clicked()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnSrvReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSrvStop</sender>
   <signal>clicked()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnSrvStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSrvStart</sender>
   <signal>clicked()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnSrvStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBDAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnAddNewQuote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBDSubmit</sender>
   <signal>clicked()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnSubmit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionO_programie</sender>
   <signal>activated()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetuj</sender>
   <signal>activated()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnSrvReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStart</sender>
   <signal>activated()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnSrvStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>activated()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnSrvStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetuj_baz_danych</sender>
   <signal>activated()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnResetDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDel</sender>
   <signal>clicked()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnDel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOtw_rz_plik_log_w_serwera</sender>
   <signal>activated()</signal>
   <receiver>MainServerWindow</receiver>
   <slot>btnOpenLogFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnSrvReset()</slot>
  <slot>btnSrvStop()</slot>
  <slot>btnSrvStart()</slot>
  <slot>btnAddNewQuote()</slot>
  <slot>btnSubmit()</slot>
  <slot>btnAbout()</slot>
  <slot>btnResetDB()</slot>
  <slot>btnDel()</slot>
  <slot>btnOpenLogFile()</slot>
 </slots>
</ui>
